app/services/flow/base_step.rb

```diff
@@ -35,6 +35,10 @@ def extra_view_variables
       {}
     end
 
+    def url_options
```

Comment:
Similar to https://github.com/18F/identity-idp/pull/4600/files#r562232318, but for flow steps